src/applications/personalization/dashboard-2/tests/e2e/messaging-error.cypress.spec.js

```diff
@@ -21,7 +21,7 @@ describe('MyVA Dashboard - Messaging', () => {
       cy.intercept('GET', '/v0/messaging/health/folders/0', ERROR_400);
```

Comment:
@ddzz I figured out the issue. In Cypress 6.9.1, this intercept worked for both the `GET /v0/messaging/health/folders/0` and also the `GET /v0/messaging/health/folders/0/messages` endpoints. Both are used by the component under test.

But in Cypress 7.3, this intercept _only_ mocks the `GET /v0/messaging/health/folders/0` endpoint.

Also note that this mocked error won't really be treated as an error since we didn't set the status code. This mock should look like this:
```
cy.intercept('GET', '/v0/messaging/health/folders/0', {
    statusCode: 400,
    body: ERROR_400,
});
```

So, what happens in Cypress 7.3 is the call to `/v0/messaging/health/folders/0/messages` fails with a 4XX because it isn't mocked at all. And, sure enough, our error handling code is not correct. So, thanks Cypress for revealing a real issue. I'm opening a PR to fix our error handling code. Feel free to leave this test skipped.

Reply:
Thanks for looking into this! I suspected it was a bug, thanks for confirming.